### PR TITLE
Add example of how to use integer values in `postBuild.substitute`

### PR DESCRIPTION
**Pull request by stefanprodan**
Fix: https://github.com/fluxcd/flux2/issues/1839

### CODE REVIEW

File: docs/spec/v1beta2/kustomization.md [hunk 1]
```diff
@@ -738,6 +738,31 @@ which will print out `${var}`.
...
+metadata:
+  name: apps
+spec:
+  ...
+  postBuild:
+    substitute:
+      AWS_ACCOUNT_ID: "71111111117"
+```
+
+For the integer to be replaced as string, the variable needs to be escaped like so:
```
Comment (hiddeco) [inline docs/spec/v1beta2/kustomization.md:755]:
```suggestion
For this integer to be replaced as string, the variable needs to be escaped:
```

**Comment by stefanprodan** [comment]
Well this doesn't actually work, on my test cluster the integer is double quoted `account: '"12345"'` instead of `account: "12345"` . Not sure why people are saying this works cc @kingdonb

**Comment by kingdonb** [comment]
It definitely worked back in September 2021. I think it stopped working around 0.18

I have seen a few threads since then where I suggested this as the resolution, and the reply has consistently been this does not work anymore, there's no workaround at present.

**Comment by bogaertg** [comment]
any workaround to make it works again ?

**Pull request closed without merging**